src/utils/stream.rs

```diff
@@ -1,80 +1,111 @@
-use std::{io::{Write, stdout, stderr, Result}, time::Instant};
+use std::{
+    io::{stderr, stdout, Result, Write},
+    time::Instant,
+};
 
 /// Handles the Printing logic for the Spinner
-#[derive(Default, Copy, Clone)]
```

Comment:
This is technically a breaking change, but I don't think it's possible to implement this without removing Copy.